huaweicloud/services/ddm/resource_huaweicloud_ddm_schema.go

```diff
@@ -460,6 +473,29 @@ func buildDeleteSchemaQueryParams(d *schema.ResourceData) string {
 	return res
 }
 
+func handleOperationError(param *handleOperationParam) (bool, error) {
+	if param.err == nil {
+		return false, nil
+	}
+	if errCode, ok := param.err.(golangsdk.ErrUnexpectedResponseCode); ok && errCode.Actual == 409 {
+		var apiError ddmError
+		err := json.Unmarshal(errCode.Body, &apiError)
+		if err != nil {
+			return false, fmt.Errorf("error %s DDM %s: error format error: %s", param.operateType,
+				param.operateObj, err)
+		}
+		if apiError.ErrorCode == "DBS.200019" {
+			err = waitForInstanceRunning(param.ctx, param.d, param.cfg, param.instanceID,
```

Comment:
we don't need to wait for the instance running in the retry func

Reply:
done